api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java

```diff
@@ -337,7 +337,7 @@ public CriteriaListWithCountResponse findCriteriaByDomain(
     Page<DbCriteria> dbCriteriaPage =
         cbCriteriaDao.findCriteriaByDomainAndCodeAndStandardAndNotType(
             domain,
-            term.replaceAll("[()+\"*-]", ""),
+            term.replaceAll("[()+\"*]", ""),
```

Comment:
This replaceAll is stale and can be removed.
```suggestion
            term,
```